src/audio/Kconfig

```diff
@@ -72,6 +72,13 @@ config COMP_SRC
 	help
 	  Select for SRC component
 
+config XRUN_RAISES_AN_ERROR
+	bool "XRUN is handled and raises an error"
```

Comment:
Best to call this NOTIFY_HOST_ON_XRUN with "Notify the host via IPC that a XRUN/Glitch has occurred"